docs/style/language_specific/capitalization-en.md

```diff
@@ -0,0 +1,53 @@
+English Capitalization Standards
+---------------
+
+The English Capitalization Standards were introduced to ensure entry naming is uniform. It is to be used on Publication, Edition, and Work entries so that future users will have an easier time browsing a list of standardized submissions.
+
+Capitalization Standards
+---------------
+1. The first and last words are always capitalized.
+2. Between the first and last word of a title, **capitalize all words except:** 
+	+ [Articles:](http://grammar.about.com/od/ab/g/articleterm.htm) a, an, the
+	+ [Conjunctions:](http://partofspeech.org/conjunction) and, but, nor, or
+	+ Short [Prepositions](http://partofspeech.org/preposition) (three letters or less): as, at, by, for, in, of, on, to, but, cum, mid, off, per, qua, re, up, via -- except when used as adverbs or as an inseparable part of a verb
+	+ the [Infinitive:](https://owl.english.purdue.edu/owl/resource/627/03/) to
+
+Special Case
+---------------
+The word O'Clock is to have both "O" and "Clock" capitalized.
+
+Examples
+---------------
+1. Rule 1
+	+ *Don Quixote*
+	+ *Great Expectations*
+	+ *Animal Farm*
+2. Rule 2
+	+ *Harry Potter and the Order of the Phoenix*
+	+ *The Pursuit of Love*
+	+ *The Call of the Wild*
+	+ *Lord of the Flies*
+	+ *Pride and Prejudice*
+	+ *How to Win Friends and Influence People*
+	+ *Lake of the Long Sun*
+	+ *Better Than Before*
+	+ *The Sun Also Rises*
+	+ *There Will Be Blood*
+* [Nouns](http://partofspeech.org/noun)
+	+ *The Book Thief*
+	+ *Of Mice and Men*
+	+ *The Portrait of a Lady*
+* [Verbs](http://partofspeech.org/verb)
+	+ *Things Fell Apart*
+	+ *One Flew Over the Cuckoo’s Nest*
+	+ *As I Lay Dying*
+4. [Adjectives](http://partofspeech.org/adjective/)
```

Comment:
This should probably be a bullet point (and 5. below), since bullets are used for the other parts of speech.